WTWSMS/common/event_modifiers/MZ_event_modifiers.txt

```diff
@@ -76,4 +83,36 @@ held_djiorgwyba_feast_timer = {
 }
 held_djiorgwyba_timer = {
 	icon = 1
+}
+
+zoroastrian_honesty = {
+	icon = 12
+	monthly_character_piety = 0.40
+	church_opinion = 5
+	major = yes
+}
+
+zoroastrian_deceitfulness = {
+	icon = 12
+	monthly_character_piety = -0.40
+	church_opinion = -5
+}
+
+zoroastrian_celibacy = {
+	icon = 12
+	monthly_character_piety = -1.50
+	church_opinion = -10
+}
+
+zoroastrian_not_married = {
+	icon = 36
+	monthly_character_piety = -0.40
+	monthly_character_prestige = -0.40
+}
+
+commissioned_shahnameh_addition = {
+	icon = 12
+	monthly_character_piety = 0.30
+	zoroastrian_opinion = 10
+	monthly_character_prestige = 0.30
```

Comment:
Good that you moved over these into this file given the performance load it adds.